static/styles/main.scss

```diff
@@ -21,6 +21,7 @@
 @import 'partials/_images.scss';
 @import 'partials/_account.scss';
 @import 'partials/_success.scss';
+@import 'partials/hourglass.scss';
```

Comment:
@abdiG I still don't think this is right -- it looks like the filename you created is `_hourglass.scss` with an underscore, but this import doesn't have the `_`.